docs/installation-on-k3d.md

```diff
@@ -0,0 +1,131 @@
+   ```
+
+4. Build and push the Keda module.
+
+   ```bash
+   make module-build IMG=k3d-kyma-registry:5001/keda-manager-dev-local:0.0.1 MODULE_REGISTRY=localhost:5001/unsigned
+   ```
+
+5. Verify if the module and the manager's image are pushed to the local registry.
+
+   ```bash
+   curl localhost:5001/v2/_catalog
+   ```
+You should get a result similar to this example:
+
+   ```json
+   {"repositories":["keda-manager-dev-local","unsigned/component-descriptors/kyma-project.io/module/keda"]}
+   ```
+6. Inspect the generated module template.
+
+> **NOTE:** The following sub-steps are temporary workarounds.
+
+Edit `template.yaml` and:
+- change `target` to `control-plane`
+
+   ```yaml
+   spec:
+    target: control-plane
+    ```
+> **NOTE:** This is required in the single-cluster mode only.
+
+- change the existing repository context in `spec.descriptor.component`:
+
+   ```yaml
+   repositoryContexts:      
+     - baseUrl: k3d-kyma-registry.localhost:5000/unsigned
+       componentNameMapping: urlPath
+       type: ociRegistry
+   ```
+
+> **NOTE:** Because Pods inside the k3d cluster use the docker-internal port of the registry, it tries to resolve the registry against port 5000 instead of 5001. K3d has registry aliases, but `module-manager` is not part of k3d and does not know how to properly alias k3d-kyma-registry.localhost:5001.
+
+7. Install the modular Kyma on the k3d cluster.
+
+> **NOTE** This installs the latest versions of `module-manager` and `lifecycle-manager`.
+
+Use the `--template` flag to deploy the Keda module manifest from the beggining, or apply it using kubectl later.
+
+   ```bash
+   kyma alpha deploy --template=./template.yaml
+   ```
+Kyma installation is ready, but the module is not yet activated.
+
+   ```bash
+   kubectl get kymas.operator.kyma-project.io -A
+   ```
+You should get a result similar to the following example:
+
+   ```bash
+   NAMESPACE    NAME           STATE   AGE
+   kcp-system   default-kyma   Ready   71s
+   ```
+
+Keda Module is a known module, but not yet activated.
+
+   ```bash
+   kubectl get moduletemplates.operator.kyma-project.io -A 
+   ```
+
+You should get a result similar to the following example:
+
+   ```bash
+   NAMESPACE    NAME                  AGE
+   kcp-system   moduletemplate-keda   2m24s
+   ```
+
+8.  Give Module Manager permission to install CRD cluster-wide.
+
+> **NOTE:** `module-manager` must be able to apply CRDs to install modules. In the remote mode (with control-plane managing remote clusters) it gets an administrative kubeconfig, targeting the remote cluster to do so. But in the local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
+
+Run the following to make sure the module manager's Service Account gets an administrative role:
+
+   ```bash
+   kubectl edit clusterrole module-manager-manager-role
```

Comment:
Will this command also change?

Reply:
probabaly..  lets improve it in separate PR